src/CodeStyle/CSharp/Analyzers/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,56 @@
+Microsoft.CodeAnalysis.CSharp.Formatting.BinaryOperatorSpacingOptions
```

Comment:
@sharwell: what's the plan for a "public API" surface area in this assembly? Shouldn't this all be internal?

Reply:
(this is not a question about this particular thing but rather asking what it even means to run the analyzer on this assembly)

Reply:
The code style packages do not have a public API. Or more specifically, we always assume that there is no downstream code referencing these assemblies and are free to break them at will. We do not distribute the compiled binaries in any package which makes them available for reference/use by other code.

Reply:
@sharwell: should we just turn off the analyzers for them then?

Reply:
If they don't have a public API that we intend to support then we should turn off the analyzer and delete the shipped / unshipped files. That is the expected state for non-public API assemblies.

Reply:
That would be fine with me. However, if we plan to try and ship these later (so consolidate analyzer implementations in NuGet and VSIX) then we would need them added back.